src/Lecture1.hs

```diff
@@ -108,7 +134,11 @@ and finds a sum of the numbers inside this string.
 
 The string contains only spaces and/or numbers.
 -}
-strSum str = error "TODO"
+
+strSum :: [Char] -> Int
+strSum str =
+    let wordsStr = words str
+    in sum (map (\x -> read x :: Int) wordsStr)
```

Comment:
As you specify the top-level type signature for this function, the compiler would be able to figure out what type `read` function has in this case due to type inference, so you can skip specifying it in the body:

```suggestion
    in sum (map read wordsStr)
```